management/backup.py

```diff
@@ -14,19 +14,34 @@
 
```

Comment:
Mode of this file needs changed back to 755

Reply:
Solved with a42f5366d6baeb0f00a30e85e33847a84c0dcd3d.